src/modules/queue/components/QueueContainer.jsx

```diff
@@ -40,6 +47,11 @@ const Composed = adapt(
 		updatePlaying: <Mutation mutation={UPDATE_PLAYING} />
 	},
 	{
+		_: ({render, playlist}) => (
+			<Subscription subscription={ON_TRACK_ADDED} variables={{playlist}}>
+				{render}
+			</Subscription>
+		),
```

Comment:
This is cheeky.

In our QueueContainer we create a Subscription that listens for playlist updates. The Subscription component takes a callback that it will pass the actual data from the update that we'd typically use to render new components.

But here's the cool part: we already have a query that is reading out all the tracks from the playlist and passing it to the QueueContainer. And since Apollo will automatically update the cache with the subscription result, it will automatically update the query results.

So we don't care at all about the results of the subscription, and we can just assign it to _ and ignore it.